_search-plugins/knn/knn-index.md

```diff
@@ -368,6 +368,7 @@ Setting | Default | Updatable | Description
 :--- | :--- | :--- | :---
 `index.knn` | false | false | Whether the index should build native library indexes for the `knn_vector` fields. If set to false, the `knn_vector` fields will be stored in doc values, but approximate k-NN search functionality will be disabled.
 `index.knn.algo_param.ef_search` | 100 | true | The size of the dynamic list used during k-NN searches. Higher values result in more accurate but slower searches. Only available for NMSLIB.
+`index.knn.advanced.approximate_threshold` | 15000   | true      | This threshold defines the minimum number of vectors required in segment, before creating specialized vector data structures for approximate search. Set `-1` to disable building vector data structures and `0` to build always.
```

Comment:
should we mention somewhere that this setting change will only be applied to indices >= 2.18 version of Opensearch?

Reply:
This will be merged in 2.18 and will be available going forward ( following versions)

Reply:
So you are saying we are relying of the documentation version to expose this information. make sense.